### DISCUSSION TITLE
Application not processing

### DISCUSSION THREAD

**Discussion by FrostBiteX09** [community/community#104718] [GitHub Education] [answered]
### Hi everyone,

Hi,
I am applying for the student benefits through their application. It asks me to upload an image of my student ID. However, when I do so the page refreshes and tells me to submit it again. I have been submitting the same student ID picture for the past two days and it keeps refreshing. I have tried on devices too and still the same result. I am not sure what I am doing wrong or that it could be with the website. If there is any solutions to this, please let me know. Thank you!

**Comment by JCCourt** [comment, marked as answer]
Hi FrostBiteX09

Welcome to Github!

* Apply via smartphone 

I think this has been a known issue for a while now. I also had the same problem uploading my files via PC. But once I tried applying on my phone I had no problems. If for whatever reason smartphone doesn't work, refer to the [supported browsers page](https://docs.github.com/en/get-started/using-github/supported-browsers)  for trouble shooting. 

If this helped. Please "Mark as Answered" so future users can see this information to help them too 😊

-- JCCOURT

**Reply by FrostBiteX09** [reply]
Thank you. I tried on my phone but it didn't work but it did on my iPad. I appreciate your help!

**Comment by RahulMohapatra1** [comment]
Hi everyone,
Hi,
I applied for the student benefits through their application. It asks me to upload an image of my student ID.  I did upload all the required documents. But since 4th Feb it is showing it has been approved bt i am unable to use the benefits. Please solve the query. Thank you!

**Reply by JCCourt** [reply]
It takes 72 hours, So either today or tomorrow you should get them.
